addons/addons.xml

```diff
@@ -17,6 +17,11 @@
     <bugtracker>https://github.com/nitrous24/Lua/issues?page=1</bugtracker>
   </addon>
   <addon>
+    <name>AnnounceTarget</name>
+    <author>JoshK6656, Sechs</author>
+    <description>Creates special manual or automatic chat messages conveying advanced information about some specific targets</description>
```

Comment:
Please also add a bugtracker (can use [Windower Lua's issue tracker](https://github.com/Windower/Lua/issues)) and a support site (can use Windower's IRC channel, see some other addons that did that).